parse.go

```diff
@@ -38,45 +37,58 @@ func ParseSource(data interface{}) (*ast.File, *token.FileSet, error) {
 
 // ParseAndTypeCheckFile parses and type-checks the given file, and returns everything interesting about the file.
 // If a fatal error is encountered the error return argument is not nil.
-func ParseAndTypeCheckFile(file string) (*ast.File, *token.FileSet, *types.Package, *types.Info, error) {
+func ParseAndTypeCheckFile(file string, flags ...string) (*ast.File, *token.FileSet, *types.Package, *types.Info, error) {
 	fileAbs, err := filepath.Abs(file)
+
 	if err != nil {
-		return nil, nil, nil, nil, fmt.Errorf("Could not absolute the file path of %q: %v", file, err)
+		return nil, nil, nil, nil, fmt.Errorf("could not absolute the file path of %q: %v", file, err)
 	}
+
 	dir := filepath.Dir(fileAbs)
 
-	buildPkg, err := build.ImportDir(dir, build.FindOnly)
-	if err != nil {
-		return nil, nil, nil, nil, fmt.Errorf("Could not create build package of %q: %v", file, err)
-	}
+	//buildPkg, err := build.ImportDir(dir, build.FindOnly)
+	//
+	//if err != nil {
+	//	return nil, nil, nil, nil, fmt.Errorf("could not create build package of %q: %v", file, err)
```

Comment:
Are this comments  useful?

Reply:
Of course not.
I'm currently testing it. After that I will clean up everything and remove [WIP] from the header :)

Reply:
Is the project still alive?